plugins/discovery-gce/qa/gce/build.gradle

```diff
@@ -42,26 +40,26 @@ restResources {
 
 /** A task to start the GCEFixture which emulates a GCE service **/
 task gceFixture(type: AntFixture) {
-  dependsOn compileTestJava
-  env 'CLASSPATH', "${-> project.sourceSets.test.runtimeClasspath.asPath}"
+  dependsOn compileYamlRestTestJava
```

Comment:
Same as above, we should depending on the source set runtime classpath here, not just the compile task.

Reply:
done https://github.com/elastic/elasticsearch/pull/60084/commits/01114f9eca06b6d834cf5c85bdc93bdc80faeb9f